js/central.js.php

```diff
@@ -49,24 +49,14 @@
             if ($('.nav-link.active:contains($locale_group_view)').length == 0) {
                return;
             }
-
-            // get central list for plugin and insert in group tab
-            $(".masonry_grid").each(function(){
-               var masonry_id = $(this).attr('id');
-
-               if (this.innerHTML.indexOf('escalade_block') < 0) {
-                  //prepare a span element to load new elements
-                  $(this).prepend("<div class='grid-item col-xl-6 col-xxl-4'><div class='card' id='escalade_block'></div></div>");
-
-                  //ajax request
-                  $("#escalade_block").load(CFG_GLPI.root_doc+"/"+GLPI_PLUGINS_PATH.escalade+'/ajax/central.php', function() {
-                     if ($("#escalade_block").html() == "") {
-                        $("#escalade_block").closest('.grid-item').remove();
-                     } else {
-                        var msnry = new Masonry('#'+masonry_id);
-                        msnry.layout();
-                     }
-                  });
+            var container = $(".masonry_grid");
+            $.ajax({ type: "GET",
+               url: CFG_GLPI.root_doc+"/"+GLPI_PLUGINS_PATH.escalade+'/ajax/central.php',
+               async: false,
+               success : function(text)
+               {
+                  container.append(text);
+                  window.msnry = new Masonry('.masonry_grid');
```

Comment:
1. Previous code was checking if response was empty before trying to append it. This can happens when there is no data to shown in the widget.
2. Is this normal to redefine `window.msnry` ?
3. Is this notmal to remove call to `msnry.layout();` ?

Reply:
If i keep the old code the widgets are badly placed, I have two widgets on top of each other
![image](https://user-images.githubusercontent.com/7335054/197695165-623843e2-7759-430a-9c02-96898fb20b7f.png)

With my method
All widgets are displayed and visible
![image](https://user-images.githubusercontent.com/7335054/197695206-ba8bbec1-e385-4a21-9d4c-c0cbd4c6961a.png)

I added the check on the ajax data received